tests/e2e/DropDownFilter/DropDownFilter.spec.ts

```diff
@@ -10,4 +11,16 @@ describe("Drop down filter", () => {
         const itemValue = defaultFilter.listViewFirstItem.getHTML();
         expect(itemValue).toContain(dropDownTestValue);
     });
+    it("should filter using multiple type of filters", () => {
+        browser.url("/p/index");
+
+        const searchBar = $(".search-bar input");
+        searchBar.waitForDisplayed();
+        searchBar.setValue(textFilterValue);
+        browser.waitUntil(() => {
+            return defaultFilter.listViewList.length === 1;
+        }, 5000, "Wait for single item.");
+        const itemValue = defaultFilter.listViewFirstItem.getHTML();
+        expect(itemValue).toContain(textFilterValue);
```

Comment:
am I missing something, doesn't this only use the searchbar?

Reply:
Yes, this page is already using a drop down filter so I am combining with the built-in search.

Reply:
Does it make sense to also use the dropdown filter in this test? so we use the choose something from the dropdown and also search something?

Reply:
Yeah! I already doing that, this page has the dropdown filters and text filter defined by default. However the only filter I am missing here is the checkbox filter, I will add it as well.